subiquity/server/mounter.py

```diff
@@ -66,6 +68,7 @@ def write(self, path, content):
 @attr.s(auto_attribs=True, kw_only=True)
 class Mountpoint(_MountBase):
     mountpoint: str
+    create: bool
```

Comment:
I read "create" as an instruction here, but really it's a record of whether the mountpoint was created by the mount code, right? so "created" would at a minimum be a better name. Also it should probably default to False?